.travis.yml

```diff
@@ -57,6 +58,9 @@ _install_backend: &_install_backend
     - mkdir -p $TRACIM_SESSION__LOCK_DIR
     - mkdir -p $TRACIM_CALDAV__RADICALE__STORAGE__FILESYSTEM_FOLDER
     - ls -R /tmp/test
+    # INFO - S.G - 2020-12-30 - needed to avoid an error with "docker login"
```

Comment:
Comment here is not clear enough for me, some more explaining or a link to a bug, may be useful here. 
I do want to know if this is a "workaround" (it look like it is) or a normal thing to do in such context (docker).

If we do not have more information about it, it may be difficult, if we decide to update docker Debian version to know if this code is always relevant.